pkg/image/validation/kubernetes_test.go

```diff
@@ -1084,192 +1079,90 @@ func TestValidateAdditionalArtifacts(t *testing.T) {
 	}
 }
 
-func TestValidateNodes(t *testing.T) {
+func TestValidateNetwork(t *testing.T) {
```

Comment:
Hey Danial, glad to see that most of my code has been reused :)
I saw you re-wrote and simplified this function, which on the one hand is good, but has a much smaller coverage. The one I wrote was carefully calibrated around the IsGlobalUnicast() check (and internal sub-checks), so that the different cases of non-global unicast values were sufficiently tested. I think it make sense to include a couple of additional malformed IPs (e.g. 500.168.1.1), but I'd bring back the cases from my initial work as it's just a matter of copying and pasting.